internal/aws/client/client.go

```diff
@@ -0,0 +1,65 @@
+package client
+
+import (
+	"context"
+	"github.com/Falco-Talon/falco-talon/utils"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+)
+
+type AWSClient struct {
+	cfg aws.Config
+
+	mu           sync.Mutex // Protects the fields below
+	lambdaClient *lambda.Client
+	//s3Client     *s3.Client
+}
+
+var (
+	awsClient *AWSClient
+	once      sync.Once
+)
+
+func Init() error {
+	var err error
+	once.Do(func() {
+		cfg, err := config.LoadDefaultConfig(context.TODO())
```

Comment:
we should provide more settings, see what's available for falcosidekick: https://github.com/falcosecurity/falcosidekick/blob/master/config_example.yaml#L150,L156

Reply:
Implemented in my last commit with all the options available in Sidekick.